lib/components/term.js

```diff
@@ -177,7 +177,20 @@ export default class Term extends Component {
   }
 
   clear() {
-    this.term.onVTKeystroke('\f');
+    // Command Running
+    if (this.term.getCursorColumn() === 0) {
```

Comment:
Ah, yes you are right thats a big issue. Do you know of any other ways to
detect if a command is running?